homeassistant/components/light/xiaomi_miio.py

```diff
@@ -20,8 +20,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-DEFAULT_NAME = 'Xiaomi Philips Light'
-PLATFORM = 'xiaomi_philipslight'
+DEFAULT_NAME = 'Xiaomi Miio Light'
```

Comment:
@syssi is this ok too?

Reply:
We should stick to "Xiaomi Philips Light" for the moment. There are no other lights available at the moment and we should try to not mix-up with the Xiaomi Yeelights (another component + open developer api)